app/models/metasploit/cache/module/target/platform.rb

```diff
@@ -7,19 +7,22 @@ class Metasploit::Cache::Module::Target::Platform < ActiveRecord::Base
   # Associations
   #
 
-  # @!attribute module_target
-  #   The module target that supports {#platform}.
-  #
-  #   @return [Metasploit::Cache::Module::Target]
+  # The module target that supports {#platform}.
   belongs_to :module_target, class_name: 'Metasploit::Cache::Module::Target', inverse_of: :target_platforms
 
-  # @!attribute platform
-  #   The platform supported by the {#module_target}.
-  #
-  #   @return [Metasploit::Cache::Platform]
+  # The platform supported by the {#module_target}.
   belongs_to :platform, class_name: 'Metasploit::Cache::Platform', inverse_of: :target_platforms
 
   #
+  # Attributes
+  #
+
+  # @!method platform_id
```

Comment:
Is there a reason these aren't `@!attribute`s? Using `@!method` will not create a setter.